Tasks/AzureFunctionOnKubernetesV0/src/kubernetesConnection.ts

```diff
@@ -0,0 +1,30 @@
+import * as tl from 'azure-pipelines-task-lib/task';
```

Comment:
This is duplicate

Reply:
Moved the logic to kubernetes-common-v2.